tensorflow/core/ops/cudnn_rnn_ops.cc

```diff
@@ -103,6 +103,49 @@ REGISTER_OP("CudnnRNN")
       return Status::OK();
     });
 
+REGISTER_OP("CudnnRNNVarSeqLen")
```

Comment:
```suggestion
REGISTER_OP("_CudnnRNNV3")
```
The `_` prefix suggests it's not public yet. Also please take a look at `CudnnRNNV2` signature and add a new output `host_reserved` for storing auto-tune outputs. (autotune isn't supported for cudnn 7.3 when using unpacked IO, but it probably would be.)